docs/rules/capitalized-comments.md

```diff
@@ -0,0 +1,240 @@
+# enforce or disallow capitalization of the first letter of a comment (capitalized-comments)
+
+The purpose of this rule is to enforce consistent casing of the first word of every comment in your codebase.
```

Comment:
This section should be about the concept that the rule addresses, not about the rule itself. See, for example, http://eslint.org/docs/rules/brace-style